### PR TITLE
Refactor indexL1InfoRoot to l1InfoTreeLeafCount

### PR DESCRIPTION
**Pull request by ignasirv**

### CODE REVIEW

File: contracts/v2/consensus/validium/PolygonValidiumEtrog.sol [hunk 1]
```diff
@@ -90,7 +90,7 @@ contract PolygonValidiumEtrog is PolygonRollupBaseEtrog, IPolygonValidium {
      */
     function sequenceBatchesValidium(
         ValidiumBatchData[] calldata batches,
-        uint32 indexL1InfoRoot,
+        uint32 l1InfoRootLeafcount,
```
[the remark this answers is not in the document]
Comment (krlosMata) [reply contracts/v2/consensus/validium/PolygonValidiumEtrog.sol:93]: this applies for all namings

[the remark this answers is not in the document]
Comment (krlosMata) [reply contracts/v2/consensus/validium/PolygonValidiumEtrog.sol:93]: maybe it is better `l1InfoTreeLeafCount` ?
instead of `root`

**Review by krlosMata — CHANGES_REQUESTED** [review]

File: contracts/v2/consensus/validium/PolygonValidiumEtrog.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (ignasirv) [reply contracts/v2/consensus/validium/PolygonValidiumEtrog.sol:93]: Yes it is!

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Pull request merged**